project.php

```diff
@@ -96,7 +96,12 @@
 }
 
 if ($user_is_logged_in) {
-    upi_set_t_latest_home_visit($pguser, $project->projectid, time());
+    upi_set_t_latest_home_visit(
+        $pguser,
+        $project->projectid,
+        time(),
+        $project->PPer_is_current_user or $project->PPVer_is_current_user
```

Comment:
*minor* any reason for `or` vs `||`? I have never seen `or` before, but php isn't my expertise.

Reply:
This is probably my python coming out. I think we've used `||` primarily before although it looks like [both work](https://www.php.net/manual/en/language.operators.logical.php). I'll see what we've done in other parts of the code and mirror that.

Reply:
My understanding is that they have different precedence.

Reply:
The code seems to use both and I think `or` is clearer so let's keep it like this.